generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java

```diff
@@ -20,14 +18,7 @@
      * @param app the spring application
      */
     public static void addDefaultProfile(SpringApplication app) {
-        Map<String, Object> defProperties =  new HashMap<>();
-        /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
-        defProperties.put(SPRING_PROFILE_DEFAULT, Constants.SPRING_PROFILE_DEVELOPMENT);
-        app.setDefaultProperties(defProperties);
+        app.setAdditionalProfiles(Constants.SPRING_PROFILE_DEVELOPMENT, Constants.SPRING_PROFILE_SWAGGER);
```

Comment:
There was an issue with include, I had a bug opened in spring and its fixed. So it would work once we upgrade to 1.4.1 but still the other issue with `bootstrap-{profile}.yml` files are there